.buildscript/check_git_clean.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/sh -eu
```

Comment:
```suggestion
#!/bin/sh -eux
```
So we can see the output as it runs